foundation/foundation-upgrade/src/test/java/com/boozallen/aissemble/upgrade/migration/v1_9_0/AirflowDockerfileMigrationSteps.java

```diff
@@ -0,0 +1,67 @@
+package com.boozallen.aissemble.upgrade.migration.v1_9_0;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.boozallen.aissemble.upgrade.migration.AbstractAissembleMigration;
+import com.boozallen.aissemble.upgrade.migration.AbstractMigrationTest;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.apache.commons.io.FileUtils;
+import org.technologybrewery.baton.BatonException;
+
+import java.io.File;
+
+import static org.junit.Assert.assertTrue;
+
+public class AirflowDockerfileMigrationSteps extends AbstractMigrationTest {
+    private AbstractAissembleMigration migration;
+    private File validatedFile;
+
+    private static Boolean validateMigration(File original, File migrated) {
```

Comment:
A: We should just move this up to the abstract test.  We've essentially fully adopted this method of test outcome verification and so don't need to rewrite for each migration.